AreaofRect.java

```diff
@@ -0,0 +1,18 @@
+public class AreaofRect{
+
+    public static int perimeter(double length,float width){
+
+        double peri= 2*(length+width);
+        return (int) peri;
+    }
+    public static void main(String[]args){
+        double length= 5.23d;
+        float width=9.73f;
+        int result= perimeter(length,width);
```

Comment:
where you invoked this perimeter function.

Reply:
line 11